po/fr.po

```diff
@@ -5026,7 +5026,7 @@ msgstr "nom de fichier"
 
 #: ../src/common/collection.c:612 ../src/common/collection.c:667
 msgid "capture time"
-msgstr "heure de capture"
+msgstr "date de capture"
```

Comment:
this one is confusion, it should be `heure` or `date/heure,` but I think just `heure` looks better.